src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +719,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
```

Comment:
It feels like we should be looking for a constructor of a specific shape. I think we should have a WellknownMember for it and that is how it should be looked-up, using special helper.